src/sage/combinat/designs/difference_matrices.py

```diff
@@ -26,7 +26,7 @@ def find_product_decomposition(g, k, lmbda=1):
 
     INPUT:
 
-    - ``g,k,lmbda`` -- integers, parameters of the difference matrix
+    - ``g``, ``k``, ``lmbda`` -- integers, parameters of the difference matrix
 
```

Comment:
`lmbda` seems to be a systematic typo throughout this method. It should be `lambda`. You may be acute on this misspelling :-)

It is also okay not to fix this if you will.

Reply:
I had to revert these changes because `lambda` is a reserved keyword.

Reply:
Right. That was why :-) 

Sorry about this.